projects/plugins/protect/src/class-status.php

```diff
@@ -75,6 +75,10 @@ public static function get_status() {
 			$status = self::get_from_options();
 		}
 
+		if ( isset( $status->last_checked ) ) {
+			$status->last_checked = get_date_from_gmt( $status->last_checked );
```

Comment:
Overwriting the same property means that if we call `get_status()` twice, the time zone will be of again, won't it? Or do we rely on the cache above to avoid that problem?